packages/react/src/utils/MockDateWrapper.tsx

```diff
@@ -0,0 +1,13 @@
+import mockdate from 'mockdate';
+import { ReactNode, useEffect } from 'react';
+
+export function MockDateWrapper({ children }: { children: ReactNode }): JSX.Element {
+  useEffect(() => {
+    mockdate.set('2023-05-05T00:00:00.000Z');
```

Comment:
Any objections to using `jest.setSystemTime(new Date('2017-01-01'))` instead of new dependency?

Reply:
Unfortunately `jest` isn't available in storybook since storybook isn't in the test environment. HOw do you feel about making `sinon` an official top-level dependency of `@medplum/react` since it's already being install elsewhere an an indirect dependency?

Reply:
ahh, right